report.php

```diff
@@ -5,8 +5,69 @@
  */
 
 require __DIR__ . '/functions.php';
+require __DIR__ . '/vendor/autoload.php';
 
 // Check required environment variables
 check_required_env();
 
-// @todo process phpunit test run results; send to WordPress.org
+$WPT_SSH_CONNECT = getenv( 'WPT_SSH_CONNECT' );
+$WPT_TEST_DIR = getenv( 'WPT_TEST_DIR' );
+$WPT_PREPARE_DIR = getenv( 'WPT_PREPARE_DIR' );
+$WPT_SSH_OPTIONS = getenv( 'WPT_SSH_OPTIONS' );
+$WPT_REPORT_API_KEY = getenv( 'WPT_REPORT_API_KEY' );
+
+log_message('Getting SVN Revision');
+$rev = exec('git -C ' . escapeshellarg( $WPT_PREPARE_DIR ) . ' log -1 --pretty=%B | grep "git-svn-id:" | cut -d " " -f 2 | cut -d "@" -f 2');
+
+log_message('Copying junit.xml results');
+$junit_location = escapeshellarg( $WPT_TEST_DIR ) . '/tests/phpunit/build/logs/junit.xml';
+
+if ( false !== $WPT_SSH_CONNECT ) {
+	$junit_location = '-e "ssh ' . $WPT_SSH_OPTIONS . '" ' . escapeshellarg( $WPT_SSH_CONNECT . ':' . $junit_location );
+}
+
+$junit_exec = 'rsync -rv ' . $junit_location . ' ./';
+perform_operations( array(
+	$junit_exec,
+) );
+
+log_message( 'Processing and uploading junit.xml' );
+
+$xml = file_get_contents( './junit.xml' );
+$results = process_junit_xml( $xml );
+
+$meta = array(
+	'php_version' => phpversion(),
```

Comment:
Yes it does, we'll need to find a way to get that, probably during the testing process.